spec/integration/mutant/isolation_spec.rb

```diff
@@ -0,0 +1,7 @@
+RSpec.describe Mutant::Isolation::Fork, mutant: false do
+  specify do
+    a = 1
+    Mutant::Config::DEFAULT.isolation.call { a = 2 }
```

Comment:
It feels to be abit redundant to re-specify the start state where it is specified in the same block. the `.from(1)` does not add anything to the strictness of the statement. Also the value of `a` does not matter at all as the only important semantic here is the unchanged value.